src/calculateRentalCost.js

```diff
@@ -28,7 +28,8 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return (days >= 7) ? (days * 40) - 50
+    : (days >= 3) ? (days * 40) - 20 : days * 40;
```

Comment:
It works, but it's harder to read and understand. In my opinion it will be better to write it using if.

Reply:
agree with Kirill, redo it with `if's`, nested ternaries are evil